### ISSUE TITLE
deselect the selection of folder when selecting the files on that folder

### ISSUE THREAD

**Issue by hvlxh** [microsoft/vscode#181694]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.78.0
- OS Version: Windows 10

*title* unless they select that folder again, this is even happening source control, etc

**Comment by lramos15** [comment]
So you're not holding shift here or anything?

**Comment by hvlxh** [comment]
yes im but look at source control section. i can even select the unselection

**Comment by lramos15** [comment]
This looks like it is working as intended to me, I'm not sure what I'm missing. What cases does the current implementation prevent?

**Comment by hvlxh** [comment]
it lights that unselectable

**Comment by lramos15** [comment]
I still don't understand. What does the bug break?
